Doc/whatsnew/3.11.rst

```diff
@@ -262,15 +272,20 @@ See :pep:`655` for more details.
 (Contributed by David Foster and Jelle Zijlstra in :issue:`47087`. PEP
 written by David Foster.)
 
+
+.. _whatsnew311-pep673:
+
 PEP 673: ``Self`` type
 ----------------------
 
 The new :data:`~typing.Self` annotation provides a simple and intuitive
 way to annotate methods that return an instance of their class. This
-behaves the same as the :data:`~typing.TypeVar`-based approach specified
-in :pep:`484` but is more concise and easier to follow.
+behaves the same as the :data:`~typing.TypeVar`-based approach
+:pep:`specified in PEP 484<484#annotating-instance-and-class-methods>`,
+but is more concise and easier to follow.
 
-Common use cases include alternative constructors provided as classmethods
+Common use cases include alternative constructors provided as
+:func:`classmethod <classmethod>`\s,
```

Comment:
```suggestion
:func:`classmethods <classmethod>`,
```

Reply:
I initially had this, but it results in the whole word `classmethods` being formatted in the literal font, which is not correct—`classmethods` is not the literal name of the linked function as the literal formatting incorrectly implies. What would be better is to just link to it as a regular prose word, like a `:ref:` (which is what is more appropriate here anyway), but that would require adding a dedicated inbound ref target label just for this, which I'm not sure is worth it. Thoughts?

Reply:
Your rationale makes sense; let's stick with your version in that case